src/api/AuthServer/AuthServer/Exceptions/NotExistingUserException.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace AuthServer.Exceptions
+{
+    public class NotExistingUserException : Exception
+    {
+        public NotExistingUserException(string message) : base(message)
```

Comment:
Can apply the same suggestions as for `AlreadyExistingUserException`, using 'email'